packages/payment-detection/src/declarative.ts

```diff
@@ -149,6 +149,7 @@ export default class PaymentNetworkDeclarative
           amount: parameters.amount,
           name: PaymentTypes.EVENTS_NAMES.PAYMENT,
           parameters: {
+            from: data.from,
             note: parameters.note,
```

Comment:
```suggestion
            note: parameters.note,
            from: data.from,
```
Just to be consistent with the data from the code below